ci/github-script/merge.js

```diff
@@ -40,24 +24,45 @@ async function runChecklist({ github, context, pull_request, maintainers }) {
         .reduce((acc, cur) => acc?.intersection(cur) ?? cur)
 
   const checklist = {
-    'PR targets one of the allowed branches: master, staging, staging-next.': [
-      'master',
-      'staging',
-      'staging-next',
-    ].includes(pull_request.base.ref),
-    'PR touches only files in `pkgs/by-name/`.': files.every(({ filename }) =>
-      filename.startsWith('pkgs/by-name/'),
-    ),
-    'PR authored by r-ryantm or committer.':
-      pull_request.user.login === 'r-ryantm' ||
-      (await committers).has(pull_request.user.id),
-    'PR has maintainers eligible for merge.': eligible.size > 0,
+    'PR targets a development branch.': classify(
```

Comment:
This should link to the relevant docs

```suggestion
    'PR targets a [development branch](../../blob/-/ci/README.md#branch-classification).': classify(
```